src/main/java/uk/gov/hmcts/reform/userprofileapi/resource/UpdateUserProfileData.java

```diff
@@ -15,10 +17,15 @@
 @Setter
 public class UpdateUserProfileData implements RequestData {
 
+    @JsonIgnore
+    private static final String NAME_FORMAT_REGEX = "^[A-Za-z'-]+$";
+
     private String email;
 
+    @Pattern(regexp = NAME_FORMAT_REGEX, message = "First name must only consist of Letters aA - zZ and the following special characters ' and -")
```

Comment:
doesn't it requires any junits ?

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/257/commits/c70410ae3d04305fb1b57a5ea3674124f69e8e25